README.md

```diff
@@ -10,24 +10,32 @@
  * Install a recent version of [Docker](https://docs.docker.com/installation/#installation) for your platform.
  * Install [docker-compose](https://docs.docker.com/compose/install/). In addition to the `curl` command they list, you can also install it from [homebrew](http://brew.sh/) or [pip](https://pypi.python.org/pypi/docker-compose/1.3.0rc1).
 
-If you're using `boot2docker` on a Mac or Windows, you'll need to make sure that it's running, first. Run `boot2docker up` and then `$(boot2docker shellinit)` in each shell you'll use to interact with Docker.
+If you're using `boot2docker` on a Mac or Windows, you'll need to make sure that it's running, first. Run `boot2docker init` on first run. 
+
+Run `boot2docker up` and then `$(boot2docker shellinit)` in each shell you'll use to interact with Docker.
 
 ### Getting Started
 
-Clone this repository and `cd` into its directory. Then:
+Clone this repository and `cd` into the `integrated` directory. Then:
```

Comment:
It's very possible that somebody would choose to rename the directory when they do the clone, e.g: `git clone git@github.com:deconst/integrated deconst-integrated`